### ISSUE TITLE
Auto test suite keyboard shortcut disables "Run JUnit Test" shortcut

### ISSUE THREAD

**Issue by GoogleCodeExporter** [hansriezebos/projectusus#28]
```
What steps will reproduce the problem?
1. Open a JUnit test class
2. Press Alt-Shift-X T

What is the expected output? What do you see instead?
I expect the test to be run. 
Instead I get a hint that Alt-Shift-X T T could be used to run a JUnit test 
(which doesn't work either on a single test class, by the way).
The test is not run.

What version of the product are you using? On what operating system?
Usus 0.6 in Helios on 64bit Linux

Please provide any additional information below.
--

```

Original issue reported on code.google.com by `conrad.m...@andrena.de` on 5 Oct 2010 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
I forgot to add: when I unbind the usus shortcut, the standard shortcut works.
```

Original comment by `conrad.m...@andrena.de` on 5 Oct 2010 at 9:19

**Comment by GoogleCodeExporter** [comment]
```
Hi Conrad,

thanks for reporting this issue. We will look into it ASAP.

Regards, Marc
```

Original comment by `mphilipp82@gmail.com` on 5 Oct 2010 at 10:51
- Added labels: **Priority-Critical**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I will change the keyboard shortcut to Alt-Shift-X U which will resolve the 
conflict.
```

Original comment by `mphilipp82@gmail.com` on 5 Oct 2010 at 7:33
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.6.1
```

Original comment by `mphilipp82@gmail.com` on 5 Oct 2010 at 7:40
- Changed state: **Fixed**
